server/forms.py

```diff
@@ -419,6 +430,21 @@ class EditClientForm(ClientForm):
         description="Do not fill out or render.",
         validators=[validators.optional()])
 
+    def __init__(self, obj=None, **kwargs):
+        self.obj = obj
+        super(ClientForm, self).__init__(obj=obj, **kwargs)
+
+    def validate(self):
+        # if our validators do not pass
+        if not super(ClientForm, self).validate():
```

Comment:
Should be `EditClientForm`. Actually, this method can be omitted because it's inherited from `ClientForm`, right?